.github/workflows/twitter-together.yml

```diff
@@ -0,0 +1,41 @@
+on:
+  push:
+    branches:
+      - develop
```

Comment:
Tweeting on merge to develop feels a little weird, but I'm on board. Likely an artifact of integrating the tweet infrastructure with the docs repo rather than it's own repo.

Reply:
Yeah, I agree -- I did this mostly so we don't redeploy the prod website every time we tweet.